tests/maker.test.js

```diff
@@ -0,0 +1,38 @@
+var should = require('should');
+
+describe('maker', function ( ) {
+  var maker = require('../lib/maker')({extendedSettings: {maker: {key: '12345'}}});
+
+  //prevent any calls to iftt
+  maker.makeRequest = function noOpMakeRequest (event, eventName, callback) { callback && callback()};
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Expected an assignment or function call and instead saw an expression.](https://www.codacy.com/app/jasoncalabrese/cgm-remote-monitor/file/758176647/issues/source?bid=2449627&fileBranchId=2449700#l7)